src/symbols/Edge/Edge.tsx

```diff
@@ -93,6 +94,7 @@ export interface EdgeProps {
 
 export const Edge: FC<Partial<EdgeProps>> = ({
   sections,
+  interpolation = 'curved',
```

Comment:
The defaults need to be set below in `defaultProps` ( there is a technical reason for this ). Can you update that?

Reply:
Of course 👍